Products/CMFPlone/tests/testPortalCreation.py

```diff
@@ -439,6 +439,11 @@ def testTTWLockableProperty(self):
         self.assertEquals(True,
                           self.properties.site_properties.lock_on_ttw_edit)
 
+    def testSortOnProperty(self):
+        # site_properties should have sort_on property
+        self.assertTrue(self.properties.site_properties \
```

Comment:
While at it, even better if done like this:
```
self.assertTrue(
    self.properties.site_properties.hasProperty('sort_on'),
)
```
So diff's look better and you know when something ends, there's a nice flake8 plugin that controls this.

Although, this:
```
sort_on_property = self.properties.site_properties.hasProperty('sort_on')
self.assertTrue(sort_on_property)
```
is even better